pontoon/base/views.py

```diff
@@ -282,13 +282,14 @@ def entities(request):
 
 
 def _get_translation_values(query):
-    return query.values(
-        'locale__code',
-        'locale__name',
-        'locale__direction',
-        'locale__script',
-        'string',
-    )
+    return [{
+        'code': translation.locale.code,
+        'direction': translation.locale.direction,
+        'locale': translation.locale.name,
+        'locale_pk': translation.locale.pk,
+        'script': translation.locale.script,
+        'translation': translation.string,
+    } for translation in query]
```

Comment:
Sorry for changing my mind, but since we're refactoring code, let's make sure we do it properly. I believe it will be way less confusing if we introduce another level of nesting in the output to hold all locale-specific values:
```python
return [
    {
        'locale': {
            'code': translation.locale.code,
            'direction': translation.locale.direction,
            'name': translation.locale.name,
            'pk': translation.locale.pk,
            'script': translation.locale.script,
        },
        'translation': translation.string,
    }
    for translation in query
]
```